internal/v1/handler_blueprints_test.go

```diff
@@ -114,6 +129,12 @@ func TestHandlers_UpdateBlueprint(t *testing.T) {
 	body["name"] = "Changing to correct body"
 	respStatusCodeNotFound, _ := tutils.PutResponseBody(t, fmt.Sprintf("http://localhost:8086/api/image-builder/v1/blueprints/%s", uuid.New()), body)
 	require.Equal(t, http.StatusNotFound, respStatusCodeNotFound)
+
+	// Test update customization users
+	body["customizations"] = map[string]interface{}{"users": []map[string]interface{}{{"name": "test", "password": "test"}}}
+	statusCode, resp = tutils.PutResponseBody(t, fmt.Sprintf("http://localhost:8086/api/image-builder/v1/blueprints/%s", result.Id), body)
+	t.Log(resp)
```

Comment:
Don't foget to remove this ;)

Reply:
removed the log from my code and also one that was above it